app/models/mixins/storage_manager_mixin.rb

```diff
@@ -0,0 +1,28 @@
+module StorageManagerMixin
```

Comment:
@hsong-rh No, it's cinder specific, not storage manager specific - that would still be too general.